android/app/generate_strings.py

```diff
@@ -79,7 +79,7 @@ def main():
     locale_dir = join(EC_ROOT, "lib/locale")
     lang_strings = defaultdict(list)
     for lang_region in [name for name in os.listdir(locale_dir)
-                        if isdir(join(locale_dir, name))]:
+                        if isdir(join(locale_dir, name)) and name != '__pycache__']:
```

Comment:
`__pycache__` appeared when I ran the desktop version of EC on my workstation, which then breaks the build process.  I imagine other developers will end up in the same situation so thought it was worth adding this exception.